hyperspy/models/edsmodel.py

```diff
@@ -317,6 +317,56 @@ def disable_xray_lines(self):
         """
         for component in self._active_xray_lines:
             component.active = False
+    def fit(self, fitter='mpfit', method='ls', grad=False,
+            bounded=True, ext_bounding=False, update_plot=False,
+            **kwargs):
+        """Fits the model to the experimental data
```

Comment:
Maybe explain here that this, by default, perform bounded optimisation using mpfit?